app/javascript/components/book.vue

```diff
@@ -4,20 +4,30 @@
       <span class="link"></span>
     </a>
     <figure class="cover">
-      <img :src="bookObj.image_url" alt="Book cover" />
+      <img :src="bookObj.image_url" :alt="this.bookCoverAltText" />
```

Comment:
To-do at some point: If the image_url is Goodreads' "no cover" one, we could fill this cover space with the book title and author instead.